src/Offer/CommandHandlers/ImportLabelsHandler.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Offer\CommandHandlers;
+
+use Broadway\CommandHandling\CommandHandler;
+use CultuurNet\UDB3\Label\LabelServiceInterface;
+use CultuurNet\UDB3\Label\ValueObjects\LabelName;
+use CultuurNet\UDB3\Offer\Commands\ImportLabels;
+use CultuurNet\UDB3\Offer\OfferRepository;
+
+final class ImportLabelsHandler implements CommandHandler
+{
+    /**
+     * @var OfferRepository
+     */
+    private $offerRepository;
+
+    /**
+     * @var LabelServiceInterface
+     */
+    private $labelService;
+
+    public function __construct(
+        OfferRepository $offerRepository,
+        LabelServiceInterface $labelService
+    ) {
+        $this->offerRepository = $offerRepository;
+        $this->labelService = $labelService;
+    }
+
+    public function handle($command): void
+    {
+        if (!($command instanceof ImportLabels)) {
+            return;
+        }
+
+        /** @var \CultuurNet\UDB3\Model\ValueObject\Taxonomy\Label\Label $importLabel */
```

Comment:
Maybe import the class? But I guess you used the FQN because it's inside a comment. (I have to find some remarks 😉)

Reply:
Yeah I can import it, but then I'll alias it as `Udb3ModelLabel`. There's no name conflict in this file, but the multiple `Label` value objects are pretty confusing. (We have three of them!)

Normally the one from udb3-models would not have been used in these domain classes, but it seems like the `ImportLabels` command accidentally uses the udb3-models Label object instead of the `CultuurNet\UDB3\Label` object

Reply:
Fixed in adbb968

Reply:
> Normally the one from udb3-models would not have been used in these domain classes, but it seems like the `ImportLabels` command accidentally uses the udb3-models Label object instead of the `CultuurNet\UDB3\Label` object

Maybe worth changing in a follow-up PR or a ticket?

Reply:
I'd like to do a larger refactoring (in chunks) to use the udb3-models value objects _everywhere_, so we can also get rid of cultuurnet/valueobjects. But I'm not sure how to create clearly scoped tickets for that that we can pick up in chunks instead of one big ticket that takes weeks.

Reply:
I think the refactoring of the abstractions in the commands & handlers is unrelated though (but yes fully agreed that the valueobjects should also be made more consistent one way or another)

Reply:
I'll try to carve out some time later this week to try and create some `TechnicalImprovement` tickets with clearly defined scopes for these refactorings